### DISCUSSION TITLE
Java heap space when save ERD as image

### DISCUSSION THREAD

**Discussion by jy1989** [dbeaver/dbeaver#11280] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/1291756/105435401-3fcb3b00-5c98-11eb-8e0b-ed091318ae40.png)

**Comment by kseniiaguzeeva** [comment]
I cannot reproduce it. Please tell me DBeaver version, your database and send log files https://github.com/dbeaver/dbeaver/wiki/Log-files.

**Comment by jy1989** [comment]
version: 7.3.3.202101161839
database: sap/hana
[.log](https://github.com/dbeaver/dbeaver/files/5864945/default.log)
[dbeaver-debug.log](https://github.com/dbeaver/dbeaver/files/5864947/dbeaver-debug.log)

**Comment by kseniiaguzeeva** [comment, marked as answer]
Most possible your ERD is quite big. We do not have an opportunity to save such objects in png. As a workaround please use SVG https://github.com/dbeaver/dbeaver/wiki/Extension-SVG-Format or GraphML format.
